packages/design-system/src/scalars/components/fragments/toggle-field/toggle-field.tsx

```diff
@@ -49,6 +49,7 @@ const ToggleRaw: React.FC<ToggleFieldProps> = ({
             disabled={disabled}
             required={required}
             description={description}
+            data-testid="label-testid"
```

Comment:
similar comment that before, why do we need to add `data-testid` here to make it work? My concern is that this attribute is going to be present in the build bundle too

Reply:
You are rigth , We can use byText, the role was unnecesary in that case, so the option getText its better option
thanks for the feedback